prototype_source/flight_recorder_tutorial.rst

```diff
@@ -48,6 +48,8 @@ Enabling Flight Recorder
 ------------------------
 There are two required environment variables to get the initial version of Flight Recorder working.
```

Comment:
Nit: Can you move this setting down to **Optional settings** section?
Or alternatively fix the comment "there are two required" environment variables.
Otherwise, customers will think that "TORCH_NCCL_DEBUG_INFO_TEMP_FILE" is the required var :-)